lib/pwsh/util.rb

```diff
@@ -87,16 +95,16 @@ def escape_quotes(text)
     # Ensure that all keys in a hash are symbols, not strings.
     #
     # @return [Hash] a hash whose keys have been converted to symbols.
-    def symbolize_hash_keys(hash)
-      if hash.is_a?(Hash)
-        hash.inject({}) do |memo, (k, v)| # rubocop:disable Style/EachWithObject
+    def symbolize_hash_keys(object)
+      if object.is_a?(Hash)
```

Comment:
It's a bit weird that this function has a different code pattern than the pascal_case and snake_case variants. Makes it hard to see whether they're all three doing the same or not.

Reply:
and if they're actually all the same shape, then you _could_ replace them with a `apply_key_mutator(f, o)` function so that

```
def pascal_case_hash_keys(object)
  apply_key_mutator(&:pascal_case, o)
end
```

etc.

Reply:
Replaced with a private method.